### ISSUE TITLE
Focus indicator contrast is less than 3:1 ratio on selected “Test” option.: A11y_Aspire website__Aspire page_Non Text Contrast

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/aspire.dev#250]
### GitHub Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #A11yWCAG; #DesktopWeb; #ChromiumEdge; #Win11; #Developer Community Partners; #SH_AspireWebsite_Web_Jan26; #WCAG1.4.11; #Non-text Contrast; 

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification."**

### Environment Details:
Application Name: Aspire website
URL: https://aspire.dev/
Chromium Edge: Version 145.0.3775.0 (Official build) dev (64-bit)
Edition: Windows 11 Enterprise Insider Preview
Version: 24H2
OS build: 26540.1001

### Repro steps:

1. Hit the URL: https://aspire.dev/.
2. TAB to "Test" or "Prod" control present under 'Local-first, production-ready'. and hit ENTER key.
3. Observe the issue that Focus indicator contrast is less than 3:1 ratio on selected “Test” or "Prod" option.

### Actual Result:
The visible focus border color over the selected “Test” option is 1.121:1 against the background which is less than 3:1.
Similar issue observed for "Prod" control.

### Expected Result:
The focus indicator (border/outline) over the selected “Test” option should have a minimum contrast ratio of 3:1 against adjacent colors.

### User Impact:
Low‑vision and keyboard‑only users may be unable to visually identify where focus is located due to insufficient contrast. This increases navigation errors and impacts independent task completion.

### Attachment:

<img width="1366" height="737" alt="Image" src="https://github.com/user-attachments/assets/371281f3-0305-4b5a-b9b5-2376c078580c" />

<img width="1366" height="731" alt="Image" src="https://github.com/user-attachments/assets/efe6dcc1-2ce7-4692-be91-46b742ae5cc4" />
